llarp/net/ip_range.hpp

```diff
@@ -24,6 +24,14 @@ namespace llarp
         : addr{std::move(address)}, netmask_bits{std::move(netmask)}
     {}
 
+    static IPRange
+    StringInit(std::string _range)
```

Comment:
any reason this isn't just a constructor?

Reply:
actually, it *is* a constructor just above and I'm blind.  probably just remove this function.

Reply:
done